conductor_api/src/conductor/admin.rs

```diff
@@ -16,8 +16,13 @@ use std::{
     fs::{self, create_dir_all},
     path::PathBuf,
     sync::{Arc, RwLock},
+    thread::sleep,
+    time::Duration,
 };
 
+/// how many milliseconds sleep all bugs under rugs
```

Comment:
lol, but also. 

Can I get an update/explainer? What's happening that we need to do this?

Reply:
This is a workaround for https://github.com/holochain/holochain-rust/issues/1446

Reply:
Ok cool, well maybe we could put that into the code itself so that there's a good record